rig.js

```diff
@@ -127,6 +127,30 @@ class RigManager {
     this.localRig.textMesh.sync();
   }
 
+  setLocalAvatarImage(avatarImage) {
+    const geometry = new THREE.CircleBufferGeometry(0.1, 32);
+    const img = new Image();
+    img.src = avatarImage;
+    img.crossOrigin = 'Anonymous';
+    img.onload = () => {
+      texture.needsUpdate = true;
+    };
+    img.onerror = err => {
+      console.warn(err.stack);
+    };
+    const texture = new THREE.Texture(img);
+    const material = new THREE.MeshBasicMaterial({
+      map: texture,
+      side: THREE.DoubleSide,
+    });
+    const avatarMesh = new THREE.Mesh(geometry, material);
+    avatarMesh.position.x = -0.5;
+    avatarMesh.position.y = -0.02;
+
+    this.localRig.textMesh.remove(this.localRig.textMesh.children[0]);
```

Comment:
I would be careful about depending on child order.

The way I usually do it is like `this.localRig.textMesh.avatarMesh = avatarMesh` and then it's by name.

Reply:
Otherwise adding a child in a different order will explode this, which would be very surprising.

Reply:
6d0860a and 537c46e